motr/st/utils/motr_rm_lock_cc_io.sh

```diff
@@ -188,25 +188,27 @@ main()
 
 	$motr_st_util_dir/m0cat $MOTR_PARAMS_2 -o $object_id -s $block_size \
                                  -c $block_count -e $dest_file'1' &
-
+	pid1=$!
 	$motr_st_util_dir/m0cat $MOTR_PARAMS_3 -o $object_id -s $block_size \
                                  -c $block_count -e $dest_file'2' &
-
-	sleep 1
+	pid2=$!
+	sleep 2
 
 	$motr_st_util_dir/m0cp $MOTR_PARAMS -o $object_id $src_file'1' \
                                 -s $block_size -c $block_count -e -u || {
 		motr_error_handling $? "Failed to update the object"
 	}
 
+	wait $pid1 $pid2
+
 	$motr_st_util_dir/m0unlink $MOTR_PARAMS -o $object_id -e || {
 		motr_error_handling $? "Failed to delete object"
 	}
 
 	diff $src_file $dest_file'1' || {
 		rc=$?
 		echo -n " Files differ, "
-		diff $src_file'1' $dest_file'1' || {
+		diff $src_file'1' $dest_file'1' && {
```

Comment:
If we get here, the "$src_file" and "$dest_file'1'" are different.
So, this checking confirms it again here?

Before this checking, add some sleep so the files can be sync?

Reply:
This check is against src_file1 not src_file.